packages/big-design/src/components/AccordionGroup/AccordionGroup.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { memo } from 'react';
+
+import { StyledAccordionGroup, StyledH2 } from './styled';
+
+export interface AccordionGroupProps {
+  header: string;
+  children?: React.ReactNode;
+}
+
+export const AccordionGroup: React.FC<AccordionGroupProps> = memo(({ header, children }) => {
+  return (
+    <StyledAccordionGroup>
+      <StyledH2>{header}</StyledH2>
```

Comment:
Why don't we just use a `Panel`?

Reply:
I didn't use `Panel` here because the padding was an issue since it's not styleable.

Reply:
Why do we need it to be styleable?

Reply:
The default panel has padding and I need the child panel component's border to extend to the full width of the parent container.

Reply:
Can we just add a wrapper component to do that using negative margins so we can just use our existing Panel component?

Reply:
Added a wrapper around the panels to resolve this 👍